rwengine/src/script/ScriptModule.hpp

```diff
@@ -175,9 +175,11 @@ class ScriptModule {
     std::map<ScriptFunctionID, ScriptFunctionMeta> functions;
 };
 
+#if !defined(_MSC_VER)
 template <>
 void ScriptModule::bind<void(const ScriptArguments&)>(
     ScriptFunctionID id, int args, void (*function)(const ScriptArguments&));
+#endif
```

Comment:
It can also be commented out on gcc 6.3.1 and clang 3.8.1.
Since we are using -std=c++14, I can assume all supported compilers do not need this specialization.